x/ccv/provider/keeper/proposal_test.go

```diff
@@ -1005,32 +1005,32 @@ func TestHandleEquivocationProposal(t *testing.T) {
 	keeperParams := testkeeper.NewInMemKeeperParams(t)
 	keeper, ctx, _, mocks := testkeeper.GetProviderKeeperAndCtx(t, keeperParams)
 	tcFail := []*evidencetypes.Equivocation{
-		&evidencetypes.Equivocation{
+		{
 			Time:             time.Now(),
 			Height:           1,
 			Power:            1,
-			ConsensusAddress: "addr1",
+			ConsensusAddress: "cosmosvalcons1kswr5sq599365kcjmhgufevfps9njf43e4lwdk",
 		},
-		&evidencetypes.Equivocation{
+		{
 			Time:             time.Now(),
 			Height:           1,
 			Power:            1,
-			ConsensusAddress: "addr2",
+			ConsensusAddress: "cosmosvalcons1ezyrq65s3gshhx5585w6mpusq3xsj3ayzf4uv6",
 		},
 	}
 
 	tcPass := []*evidencetypes.Equivocation{
-		&evidencetypes.Equivocation{
+		{
 			Time:             time.Now(),
 			Height:           1,
 			Power:            1,
-			ConsensusAddress: "addr3",
+			ConsensusAddress: "cosmosvalcons1kswr5sq599365kcjmhgufevfps9njf43e4lwdk",
 		},
-		&evidencetypes.Equivocation{
+		{
 			Time:             time.Now(),
 			Height:           1,
 			Power:            1,
-			ConsensusAddress: "addr4",
+			ConsensusAddress: "cosmosvalcons1ezyrq65s3gshhx5585w6mpusq3xsj3ayzf4uv6",
 		},
 	}
```

Comment:
Hi, why `tcPass[1]` is the same test case as `tcFail[1]`

Reply:
I will refactor the test to use the same structure for each test case, good catch 👍

Reply:
Done